practice.kt

```diff
@@ -0,0 +1,29 @@
+
+////////task 3 40, 50 min
+fun main() {
+
+    var list = listOf<Int>(3,2,1,4)
+    val indices=IndicesSum(list, 5)
+    println(indices.first)
+    println(indices.second)
+    
+ }
+ fun IndicesSum(list:List<Int>, x:Int):Pair<Int,Int> {
+     var indice:Int = 0
+     var indice1:Int= 0
+outerloop@ for(i in 0..list.size-1)
```

Comment:
you can use list.indices instead of list.size-1 for best practice

Reply:
will the second loop remain 1..list.size-1 because it start with 1 or is there any other way to use list.indcies starting with 1?